src/lib/yang/alarms.lua

```diff
@@ -729,6 +749,7 @@ function selftest ()
    do_add_to_inventory({alarm_type_id='arp-resolution'}, {
       resource='nic-v4',
       has_clear=true,
+      alt_resource={'nic-v4-2'},
```

Comment:
This is OK but you still need to add a test that verifies that when 'arp-resolution' alarm is retrieved it contains an `alt_resource` attribute and that attribute is a list. Something like:

```
   -- Raising an alarm when alarms is empty, creates an alarm.
   local key = alarm_keys:fetch('nic-v4', 'arp-resolution')
   raise_alarm(key)
   local alarm = assert(state.alarm_list.alarm[key])
   assert(#alarm.alt_resource == 1)
```